templates/bot/js/command-and-response/README.md

```diff
@@ -1,3 +1,86 @@
 # Command and Response Bot
 
-For how to use the command and response bot, please refer to [this document](https://aka.ms/teamsfx-command-response).
\ No newline at end of file
+A command and response bot is an app that responds to simple commands sent in Teams chat and replies a result in meaningful ways.
+
+## Get Started
+
+Before run this app locally, make sure you have prepared these prerequisites:
+
+- Node.js (Recommended version is 14)
+- An [M365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+
+Then, you can quickly start local debugging via `F5` in VSCode. Select `Debug (Edge)` or `Debug (Chrome)` debug option of your preferred browser.
+
+## Develop
+
+This new project folder structure looks like:
+
+| Folder | Contents |
+| - | - |
+| `.fx` | Project level settings, configurations, and environment information |
+| `.vscode` | Visual Studio Code files for local debug |
+| `bot` | The bot source code |
+| `templates` |Templates for Teams app manifest and corresponding Azure resources|
+
+The core command-response implementation is in `bot/` folder, containing following content:
+
+| File / Folder | Contents |
+| - | - |
+| `src/adaptiveCards/` | Adaptive card templates |
+| `src/internal/initialize.ts(js)` | Generated initialize code for initialize the command bot |
+| `src/helloworldCommandHandler.ts` | A hello world command handler to process a helloworld command and return an adaptive card as response |
```

Comment:
ts(js)

Reply:
Fixed. Just use `.ts` for TypeScript readme, and `.js` for JavaScript readme.